captain/module.go

```diff
@@ -21,8 +21,11 @@ import (
 
 var module *modules.Module
 
+const onSPNConnectEvent = "spn connect"
```

Comment:
I would suggest using `spn connected`, as this makes it clear that the connection has been established, and is not in progress anymore.